### PR TITLE
harsh Idnits - fixes

### PR DESCRIPTION
**Pull request by gorryfair**
Fixes references.
Fixed table width

We need also to fix the Licence citation
Do not understand the indent warning (possibly this is resolved by fixing table width?)

### CODE REVIEW

File: draft-ietf-tsvwg-dscp-considerations-08.xml [hunk 1]
```diff
@@ -539,12 +541,16 @@ configured to support DiffServ. (See also <xref target="remark"></xref>).
 ]]></artwork>
 
             <postamble>Figure showing 0b000xxx DSCPs, highlighting any current assignments and whether they are affected by any known remarking behaviors. 
-		For example, ToS Precedence Bleaching of popular DSCPs AF11,21,31,41 would result in traffic being remarked with DSCP 2 in the Internet core. 
-		  * DSCP 4 has been historically used by the SSH application, following semantics which precede DiffServ<xref target="Kol10"></xref>.
+	* DSCP 4 has been historically used by the SSH application <xref target="Kol10">.</xref>.
```
Comment (ana-cc) [inline draft-ietf-tsvwg-dscp-considerations-08.xml:544]: \This line about DSCP 4 is repeated below, is this a mistake?

**Review by ana-cc — CHANGES_REQUESTED** [review]

**Comment by ana-cc** [comment]
Also, these changes should be put into a brand new file for version 09, rather than applied to our submitted version 08.

File: draft-ietf-tsvwg-dscp-considerations-08.xml (hunk 1 above)
Comment (gorryfair) [reply draft-ietf-tsvwg-dscp-considerations-08.xml:544]: Not a mistake - the legend footnote; and text both need to describe this. I'll propose better :-) text in the PR, and keep the legend.

Comment (ana-cc) [reply draft-ietf-tsvwg-dscp-considerations-08.xml:544]: Ok - once you've done this I will merge everything in a new file and we can go from there!

**Pull request merged**